src/install_host_app.sh

```diff
@@ -11,8 +11,14 @@ HOST_URL="https://github.com/passff/passff-host/releases/download/$VERSION/passf
 MANIFEST_URL="https://github.com/passff/passff-host/releases/download/$VERSION/passff.json"
 KERNEL_NAME=$(uname -s)
 
+PATH="/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin"
```

Comment:
It's a terrible idea to unconditionally replace the system's `PATH` with this value. It is definitely going to break the install script for a lot of people.

Reply:
Ah, yeah, fair enought. This one is just a reasonably clean standard path, but using the current path makes sense.